src/colorizer/Collection.ts

```diff
@@ -252,6 +253,10 @@ export default class Collection {
       collectionData.set(entry.name, entry);
     }
 
+    triggerAnalyticsEvent(AnalyticsEvent.COLLECTION_LOAD, {
+      collectionWriterVersion: collection.metadata?.writerVersion || "N/A",
```

Comment:
Could also use "v0.0.0" as a placeholder but could also be a custom Collection JSON that a user has written without using the writer scripts.